src/sass/nft-collection.scss

```diff
@@ -10,11 +10,58 @@
   &__left-side {
     width: 80%;
 
-    .label {
-      font-size: 18px;
-      font-weight: 400;
-      line-height: 22px;
-      color: #000000;
+    .nft-summary {
```

Comment:
Same here `.nft-summary .route .count .floor` don't have to be embedded into `&__left-side` they can be on the same level as it.